manifests/init.pp

```diff
@@ -33,6 +33,9 @@
 # $pulp_oauth_secret::                  OAuth secret to be used for Pulp REST interaction
 #                                       type:Optional[String]
 #
+# $pulp_max_speed::                     The maximum download speed for a Pulp task per second. (e.g. "4 Kb" or 1Gb")
+#                                       type:string
```

Comment:
We moved over to Puppet 4 types, can you change this to be `type:Optional[String]`?